besu/src/main/java/org/hyperledger/besu/controller/TransitionBesuControllerBuilder.java

```diff
@@ -145,13 +151,50 @@ protected PluginServiceFactory createAdditionalPluginServices(
     return new NoopPluginServiceFactory();
   }
 
+  @Override
+  protected EthProtocolManager createEthProtocolManager(
+      final ProtocolContext protocolContext,
+      final boolean fastSyncEnabled,
+      final TransactionPool transactionPool,
+      final EthProtocolConfiguration ethereumWireProtocolConfiguration,
+      final EthPeers ethPeers,
+      final EthContext ethContext,
+      final EthMessages ethMessages,
+      final EthScheduler scheduler,
+      final List<PeerValidator> peerValidators) {
+
+    EthProtocolManager ethProtocolManager =
+        super.createEthProtocolManager(
+            protocolContext,
+            fastSyncEnabled,
+            transactionPool,
+            ethereumWireProtocolConfiguration,
+            ethPeers,
+            ethContext,
+            ethMessages,
+            scheduler,
+            peerValidators);
+
+    ConsensusContext cc = protocolContext.getConsensusContext(ConsensusContext.class);
+    if (cc instanceof MergeContext) {
+      protocolContext
+          .getConsensusContext(MergeContext.class)
+          .observeNewIsPostMergeState(ethProtocolManager);
+      protocolContext
+          .getConsensusContext(MergeContext.class)
+          .addNewForkchoiceMessageListener(ethProtocolManager);
+    }
```

Comment:
This might be better to have in the MergeBesuControllerBuilder since the intent is for all of the Transition* classes to be dropped after the paris fork.  the Transition builder composes the pre-merge and post merge builders, so it should be functionally equivalent to have this in the merge controller builder.